packages/core/networks/42.json

```diff
@@ -1,66 +1,66 @@
 [
   {
     "contractName": "Migrations",
-    "address": "0xB851449bC030F08E0f1Aa57A0a7cAEFE73030De0"
```

Comment:
I think you should remove all changes to `42.json` from this PR, as we already support a set of contracts that are integrated into `emp-tools` for example. However, were these contracts all verified on Etherscan via your script?

Reply:
Can do, this is more to show-case the automation side of it. I can revert it back to the original contracts.

Reply:
I'll revert the contract addresses once the PR comes to a consensus. I left it here just to showcase how the process works.

Reply:
SGTM!